app/components/navbar/navbar.js

```diff
@@ -139,43 +103,13 @@ export default withTranslation()(class extends React.Component {
   };
 
   renderMiddleSection = () => {
-    var patient = this.props.patient;
-
-    if (_.isEmpty(patient)) {
-      if (
-        _.includes([
-          '/patients',
-          '/clinic-admin',
-          '/prescriptions',
-          '/clinic-invite',
-          '/clinic-profile',
-          '/clinic-workspace',
-          '/clinic-workspace/patients',
-          '/clinic-workspace/invites',
-          '/clinic-workspace/prescriptions',
-          '/clinician-edit',
-        ], this.props.currentPage) && personUtils.isClinicianAccount(this.props.user)
-      ) {
-        return (
-          <WorkspaceSwitcher api={this.props.api} trackMetric={this.props.trackMetric} />
-        );
-      }
-      return <div className="Navbar-patientSection"></div>;
+    if (personUtils.isClinicianAccount(this.props.user)) {
```

Comment:
I _believe_ we can remove all of the route-matching conditions we previously used because we are now also showing the workspace switcher in the patient pages, per the figma mocks. It would be great if I can simplify this, but I'm a little worried that there are pages I'm not aware of where we would not want to show the Workspace Switcher.

Reply:
Sounds good. If possible, I _would_ like the big condition because I think it really enhances readability, as opposed to adding a few new routes to the previous condition.

I'll make a note to QA when we move this to QA to triple check that there is no place the workspace switcher shows up that it should not.